lib/commands/dump.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const path = require( 'path' );
+const { tools } = require( '@ckeditor/ckeditor5-dev-utils' );
```

Comment:
Don't use dev-utils in mgit2. Let's avoid bigger dependencies in this tool as much as we can.